sorc/machine-setup.sh

```diff
@@ -19,7 +19,7 @@ fi
 target=""
 USERNAME=`echo $LOGNAME | awk '{ print tolower($0)'}`
 
-if [[ -d /lfs3 ]] ; then
+if [[ -d /lfs4 ]] ; then
```

Comment:
Just curious why the directory was change here.

Reply:
@GeorgeGayno-NOAA I thought /lfs3 disk is no longer available on Jet. Only /lfs1 and /lfs4.

Reply:
@BinLiu-NOAA You are correct. The /lfs3 directory is empty, but machine-setup.sh was still working. Thanks for catching this.